spec/models/category_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+describe Category do
+  it { should have_many(:videos) }
+
+  describe "#recent_videos" do
+    it "should return all videos in the category if there's less than 6." do
```

Comment:
You should not use `should` word in example messages - Read more [here](http://www.betterspecs.org/#should)